recipes/ggplot/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.9.4" %}
+
+package:
+  name: ggplot
+  version: {{ version }}
+
+source:
+  fn: ggplot-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/ggplot/ggplot-{{ version }}.tar.gz
+  md5: 0fbaa3123cb2d4b060b200a106340421
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - brewer2mpl
```

Comment:
When using `--single-version-externally-managed` you probably don't need to list the run time dependencies at build time (unless the package imports itself). Try removing all but `python` and `setuptools` and let's see if that works here.